### PR TITLE
Preparing release version 3.8.0

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
:)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,81 @@ with advance notice in the **Deprecations** section of releases.
...
+  * ``node.warn(PytestWarning("some message"))``: is now the recommended way to call this function. The warning
+    instance must be a ``PytestWarning`` or subclass  instance.
+
+  * ``node.warn("CI", "some message")``: this code/message form is now deprecated and should be converted to
+    the warning instance form above.
+
+  ``RemovedInPytest4Warning`` and ``PytestExperimentalApiWarning`` are now part of the public API and should be accessed
+  using ``pytest.RemovedInPytest4Warning`` and ``pytest.PytestExperimentalApiWarning``.
+
+
```
Comment (RonnyPfannschmidt) [inline CHANGELOG.rst]: we might want to consider splitting deprecations and removals

Comment (nicoddemus) [reply CHANGELOG.rst]: Hmm good point

File: doc/en/example/pythoncollection.rst [hunk 2]
```diff
@@ -180,18 +180,32 @@ Finding out what is collected
...
-    platform linux -- Python 3.x.y, pytest-3.x.y, py-1.x.y, pluggy-0.x.y
-    rootdir: $REGENDOC_TMPDIR, inifile: pytest.ini
-    collected 3 items
-    <Module 'CWD/pythoncollection.py'>
-      <Function 'test_function'>
-      <Class 'TestClass'>
-        <Instance '()'>
-          <Function 'test_method'>
-          <Function 'test_anothermethod'>
+    Traceback (most recent call last):
```
Comment (nicoddemus) [inline doc/en/example/pythoncollection.rst]: Hmm this is incorrect, probably some update to regendocs?

Comment (RonnyPfannschmidt) [reply doc/en/example/pythoncollection.rst]: no idea off-hand, its the only hunk in that file, i'll revert that file and sort it out later

**Comment by nicoddemus** [comment]
Closing in favor of #3940

**Comment by nicoddemus** [comment]
Weird. Perhaps can someone else give this a try as well? One just needs to checkout 69b34f7 and run `tox -e release -- 3.8.0` and see if the doc generation works well.

**Pull request closed without merging**